# Update the visibility of the jax_computation so that it can be used without
# importing all of the federated-language-jax package. There is an unexplored
# issue importing both the tensorflow-federated and federated-language-jax
# bindings and this patch avoids trying to debug that issue.

diff --git federated_language_jax/computation/BUILD federated_language_jax/computation/BUILD
index e7adb04..6d4f725 100644
--- federated_language_jax/computation/BUILD
+++ federated_language_jax/computation/BUILD
@@ -32,6 +32,7 @@ py_library(
 py_library(
     name = "jax_computation",
     srcs = ["jax_computation.py"],
+    visibility = ["//visibility:public"],
     deps = [
         ":jax_serialization",
         "@federated_language_jax_pypi//dm_tree",
